tests/tests_unit/test_cdf_tk/test_build_files.py

```diff
@@ -1,47 +0,0 @@
-from __future__ import annotations
```

Comment:
I removed this file as the test here is outdated. Now we test sequences of build deploy which includes this test, so it serves no purpose any more.